pkg/users/password-dialogs.js

```diff
@@ -207,6 +207,9 @@ export function set_password_dialog(account, current_user) {
         if (state.password != state.password_confirm)
             errors.password_confirm = _("The passwords do not match");
 
+        if (state.password.length > 256)
+            errors.password = _("Password is longer than 256 characters");
+
```

Comment:
This gets overwritten by a bad quality check below. Also, given that the test failure somehow lands on the overview page, this seems to be wired up wrongly? That error should appear inside the dialog, right?

Reply:
The fact it would get overwritten is fine in my option.
This is very corner case check and also I consider quality check to be of higher quality.

And I just realized that we have two separate `validate` methods, one for changing password and one for validating a new password when creating a new account.